src/views/Bond/components/BondModal/BondModal.tsx

```diff
@@ -158,6 +158,6 @@ const BondModal: React.VFC<{ bond: Bond }> = ({ bond }) => {
 const TokenPrice: React.VFC<{ token: Token; isInverseBond?: boolean }> = ({ token, isInverseBond }) => {
   const { data: priceToken = new DecimalBigNumber("0") } = useTokenPrice({ token, networkId: NetworkId.MAINNET });
   const { data: ohmPrice = 0 } = useOhmPrice();
-  const price = isInverseBond ? priceToken.mul(new DecimalBigNumber(ohmPrice.toString())) : priceToken;
+  const price = isInverseBond ? new DecimalBigNumber(ohmPrice.toString()) : priceToken;
   return price ? <>${price.toString({ decimals: 2, format: true, trim: false })}</> : <Skeleton width={60} />;
```

Comment:
How about we do this instead? This would force consistent display of ohmPrice and ohmPrice will match the bond table. All other methods of displaying ohmPrice use formatCurrency. I think there are some rounding errors happening because we're converting to string twice for ohmPrice .
```suggestion
  const price = isInverseBond
    ? formatCurrency(ohmPrice, 2)
    : priceToken.toString({ decimals: 2, format: true, trim: false });
  return price ? <>{price}</> : <Skeleton width={60} />;
};
```